lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+attr_reader :print_menu, :todays_menu
+
+    def initialize
```

Comment:
Your indentation has gone off here.  Ensure your text editor is configured to convert tabs to 2 spaces.